lib/rest.js

```diff
@@ -78,6 +78,16 @@ function acsRequest(apiEntryPoint, appOptions, httpMethod, cookieString, restOpt
 		restOptions.pretty_json = true;
 	}
 
+	// response_json_depth can be set as application level from appOptions
+	if (appOptions.responseJsonDepth && !restOptions.response_json_depth) {
+		if (typeof appOptions.responseJsonDepth !== 'number') {
+			return callback(new ACSError(messages.ERR_WRONG_TYPE, {
+				typeName: 'app options responseJsonDepth'
```

Comment:
It's good to check type of appOptions.responseJsonDepth, and I think it would be good enough that we just simply use ```typeName: 'responseJsonDepth'```. What do you think?
Also can you please do me a favor to add the same check for prettyJson above?